app/controllers/application_controller.rb

```diff
@@ -14,6 +14,10 @@ def set_slimmer_headers
     response.headers[Slimmer::Headers::TEMPLATE_HEADER] = "header_footer_only"
   end
 
+  def finder_base_path
+    "/#{finder_slug}"
```

Comment:
Yes, it's a bit weird. We could look at changing it, but until we move the email signup stuff into email-alert-frontend we have to handle paths with multiple components anyway.